pkgs/top-level/haskell-packages.nix

```diff
@@ -6,6 +6,9 @@ let
     "ghc6102Binary"
     "ghc704Binary"
     "ghc742Binary"
+    "ghc784Binary"
```

Comment:
This mean we can get rid of those ignored integer simple flags?

Reply:
Fixed! Thanks